app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<main>
+  <h2>Log In</h2>
+
+  <form action="/login" accept-charset="UTF-8" method="post">
```

Comment:
This creates a form and then there is another form right below - this shouldn't be needed